android/modules/filesystem/src/java/ti/modules/titanium/filesystem/ResourceHelper.java

```diff
@@ -0,0 +1,28 @@
+package ti.modules.titanium.filesystem;
```

Comment:
Maybe this should go in `org.appcelerator.titanium.util` with our other helpers?